dashboard/src/components/OperatorList/OperatorList.test.tsx

```diff
@@ -75,11 +112,41 @@ it("skips the error if the OLM is not installed", () => {
   expect(wrapper.find(OLMNotFound)).toExist();
 });
 
-it("render the operator list if the OLM is installed", () => {
+it("render the operator list with installed operators", () => {
   const wrapper = shallow(
-    <OperatorList {...defaultProps} isOLMInstalled={true} operators={[sampleOperator]} />,
+    <OperatorList
+      {...defaultProps}
+      isOLMInstalled={true}
+      operators={[sampleOperator]}
+      csvs={[sampleCSV]}
+    />,
+  );
+  expect(wrapper.find(OLMNotFound)).not.toExist();
+  expect(wrapper.find(InfoCard)).toExist();
+  // The section "Available operators" should be empty since all the ops are installed
+  expect(wrapper.find("h3").filterWhere(c => c.text() === "Installed")).toExist();
+  expect(
+    wrapper
+      .find(CardGrid)
+      .last()
+      .children(),
+  ).not.toExist();
+  expect(wrapper).toMatchSnapshot();
+});
+
+it("render the operator list with installed operators", () => {
+  const wrapper = shallow(
+    <OperatorList {...defaultProps} isOLMInstalled={true} operators={[sampleOperator]} csvs={[]} />,
   );
   expect(wrapper.find(OLMNotFound)).not.toExist();
   expect(wrapper.find(InfoCard)).toExist();
```

Comment:
It's not clear to me if this should be asserting that the "Installed" section is not present when nothing is installed. That is, should this just show the available operators without even an 'Installed" section when there are non installed?

EDIT: I see from the code and snapshot below that this is indeed the way it renders, so I guess I expected to see here something like an assertion that the h3 heading "Installed" does not exist (but does above). Then again. the snapshot does that for us - *shrug*.

Reply:
> EDIT: I see from the code and snapshot below that this is indeed the way it renders, so I guess I expected to see here something like an assertion that the h3 heading "Installed" does not exist (but does above). Then again. the snapshot does that for us - _shrug_.

That's exactly what I am doing in the lines 144 and 127

Reply:
Not sure how I missed 144 :)